driver/razercommon.h

```diff
@@ -55,10 +55,10 @@ do { \
 #define RIGHT_SIDE_LED    0x10
 #define LEFT_SIDE_LED     0x11
 
-// Some keypads have different profile LED IDs
-#define ALTERNATE_RED_PROFILE_LED   0x0E
-#define ALTERNATE_GREEN_PROFILE_LED 0x0C
-#define ALTERNATE_BLUE_PROFILE_LED  0x0D
+// Some keypads have different profile LED IDs mapped
+#define ALTERNATE_RED_PROFILE_LED   BLUE_PROFILE_LED
```

Comment:
Wasn't what I had in mind, but ok...

Reply:
I see... well, it could be done like this or just inside the `switch`, not too fused. The maintainer who clicks the Merge button can have the final say. :wink:

I originally thought to use an "alternate" variable so it's pretty obvious later that a certain range of Razer keypads followed a different "spec" / wiring. Someone poking through the source code could mistakenly assume these are the "correct" definitions for all keypads. Plus, I didn't want the `red` function referring the `blue` twice (get/set) since it confuses me enough. :sweat_smile:

Reply:
Fair enough.